packages/next/src/export/routes/app-page.ts

```diff
@@ -95,7 +95,7 @@ export async function exportAppPage(
     // instead of the standard rsc. This is because the standard rsc will
     // contain the dynamic data. We do this if any routes have PPR enabled so
     // that the cache read/write is the same.
-    else if (renderOpts.experimental.isAppPPREnabled) {
+    else if (renderOpts.experimental.isRoutePPREnabled) {
```

Comment:
I can't remember exactly, but won't the builder try to route to `.prefetch.rsc` if PPR is enabled in any capacity?  e.g. https://github.com/vercel/vercel/blob/a26cbd819964f28b3696f6fcf8b928e138186997/packages/next/src/utils.ts#L416